packages/quantic/force-app/main/default/lwc/quanticResultChildren/__tests__/quanticResultChildren.test.js

```diff
@@ -156,12 +156,8 @@ describe('c-quantic-result-children', () => {
       const childResult = element.shadowRoot.querySelector(
         selectors.quanticResult
       );
-      const noMoreChildrenMessage = element.shadowRoot.querySelector(
-        selectors.noMoreChildrenMessage
-      );
 
       expect(childResult).toBeNull();
-      expect(noMoreChildrenMessage).not.toBeNull();
```

Comment:
Doing like this you are not testing that the `noMoreChildrenMessage` is not displayed.
Like we need to test if it is displayed, we also need to test if it's not displayed by doing:
```
 expect(noMoreChildrenMessage).toBeNull();
```

Reply:
yeah I guess with the additional check in the `shouldDisplayNoChildrenMessage` it makes more sense to test that it is not shown